config/olm-install/install-resources.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: mymetallb
+  labels:
```

Comment:
What is this for?

Reply:
Leftover, OpenShift scrapes namespaces labeled with `openshift.io/cluster-monitoring: "true"`.